build.rs

```diff
@@ -54,18 +61,33 @@ fn main() {
         .expect("Couldn't write bindings!");
 }
 
-fn build_dtrace() {
-    Command::new("git")
-        .args(&[
-            "clone",
-            "https://github.com/microsoft/DTrace-on-Windows.git",
-            "target\\dtrace",
-        ])
-        .output()
-        .expect("Failed to clone dtrace");
+fn build_dtrace() -> anyhow::Result<()> {
+    let output = if !PathBuf::from(DTRACE_SRC_DIR).is_dir() {
+        Command::new("git")
+            .args(&[
+                "clone",
+                "https://github.com/microsoft/DTrace-on-Windows.git",
+                DTRACE_SRC_DIR,
```

Comment:
Don't clone anything inside root directory, use the `target` directory for any external dependencies